packages/api/core/src/api/init-scripts/init-npm.ts

```diff
@@ -25,12 +26,41 @@ export const devDeps = [
 ];
 export const exactDevDeps = ['electron'];
 
+const execAndTrimResult = (command: string) => new Promise<string>((resolve, reject) => {
+  exec(command, (err, stdout) => {
+    if (err) {
+      reject(err);
+    } else {
+      resolve(stdout.trim());
+    }
+  });
+});
```

Comment:
This would be simpler with [`util.promisify`](https://nodejs.org/api/util.html#util_util_promisify_original).